src/main/webapp/app/course/competencies/competency-management/competency-import-course.component.ts

```diff
@@ -0,0 +1,53 @@
+import { Component } from '@angular/core';
+
+import { Course, CourseForImportDTO } from 'app/entities/course.model';
+import { SortService } from 'app/shared/service/sort.service';
+import { Router } from '@angular/router';
+import { NgbActiveModal } from '@ng-bootstrap/ng-bootstrap';
+import { CourseForImportDTOPagingService } from 'app/course/course-for-import-dto-paging-service';
+import { Column, ImportComponent } from 'app/shared/import/import.component';
+
+const tableColumns: Column<Course>[] = [
```

Comment:
The `tableColumns` and `entityName` attributes are only relevant for the standard template, you don't need them when adding your own html template.

Reply:
They are still here :)

Reply:
I removed the entityName as it is not necessary. I thought the tableColumns made sense to keep it as similiar to the standard template. This would reduce confusion/effort for future refactorings imo.